app/src/main/res/layout/screen_registration_name.xml

```diff
@@ -50,10 +50,7 @@
         android:gravity="center"
         android:imeOptions="actionDone"
         android:inputType="textPersonName|textCapWords"
-        android:textAppearance="@style/Clinic.LoginInputTextAppearance">
-
-        <requestFocus />
```

Comment:
I have also seen this happen on `EnterOtpScreen` for entering OTP.
Although the issue gets resolved if you close and open the screen again.
We should check for the usages of `requestFocus` throughout the app and fix this issue.

Reply:
Good point, I'll go through the codebase and see.

I think this issue only happens in cases where the previous screen in the flow already had a keyboard open, though. If a screen is being opened without the keyboard being open, there isn't any problem.

Reply:
I checked the codebase, and we have four more usages.

1. Enter OTP screen. I ran it and verified, the problem does not happen.
2. Blood Pressure Entry sheet. I ran and verified, the problem does not happen.
3. Appointment Reminder Sheet. For some reason, this is on a `TextView`. No idea why it's there, but there is no editable field on that sheet.
4. Schedule Appointment Sheet. Same as above

I can't see any other places where this needs to be fixed at the moment.

Reply:
I can see this problem on `Enter OTP Screen` in QA atleast.
I wonder why?

Reply:
I think we can make this change on `Enter OTP screen` only.
Rest of the screens look fine.

Reply:
Done.